tests/gpflow/models/test_vgp_posterior.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pytest
+
+import gpflow
+from gpflow.models.vgp import VGP_deprecated, VGP_with_posterior
+from gpflow.posteriors import PrecomputeCacheType
+
+
+def make_models(regression_data):
+    """Helper function to create models"""
+
+    k = gpflow.kernels.Matern52()
+    likelihood = gpflow.likelihoods.Gaussian()
+
```

Comment:
minor: maybe add a test for a non-gaussian likelihood?

Reply:
Actually I might be mistaken here...because there is no training it might not make any difference

Reply:
I've added another likelihood - better safe than sorry.